owlbot.py

```diff
@@ -0,0 +1,15 @@
+import synthtool as s
```

Comment:
Actually, this is missing a license header

Reply:
Added.